libff/CMakeCache.txt

```diff
@@ -0,0 +1,568 @@
+# This is the CMakeCache file.
+# For build in directory: /Users/valar/Code/git repos/libff/libff
```

Comment:
This file should not be added. (Side note: Careful with committing local files as they leak information about the originator's machine - i.e. configuration (OS, compiler etc), file structure etc. ;) )
The `build` directory should be ignored by git already, but I assume that `cmake` was ran from the root repo (i.e. `cd $LIBFF/libff && cmake .` was ran instead of `cd $LIBFF && mkdir build && cd build && cmake ..`).

Reply:
Removed in https://github.com/scipr-lab/libff/pull/90